### PR TITLE
Return an script timeout error for execute script if timeout value re…

### PR DESCRIPTION
**Pull request by AutomatedTester**
…ached. Fixes #issue378

This adds in a wait for the script to complete executing or return an error.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/532)

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -5435,9 +5435,16 @@
  <li><p>Let <var>result</var> be the result of calling <a>execute a function body</a>,
   with arguments <var>body</var> and <var>arguments</var>.
 
+ <li><p>Wait for <a>execute a function body</a> to return
+  or for <a>session script timeout</a> milliseconds to expire,
+  whichever happens sooner.
+
+ <li><p>If the previous step completed due the <a>session script timeout</a> being reached,
+    return <a>error</a> with <a>error code</a> <a>script timeout</a>.
```
Comment (shs96c) [inline webdriver-spec.html]: Nit: indentation is inconsistent.

File: webdriver-spec.html [hunk 2]
```diff
@@ -5435,9 +5435,16 @@
  <li><p>Let <var>result</var> be the result of calling <a>execute a function body</a>,
   with arguments <var>body</var> and <var>arguments</var>.
 
+ <li><p>Wait for <a>execute a function body</a> to return
+  or for <a>session script timeout</a> milliseconds to expire,
+  whichever happens sooner.
+
+ <li><p>If the previous step completed due the <a>session script timeout</a> being reached,
```
Comment (shs96c) [inline webdriver-spec.html]: s/due the/due to the/

**Comment by shs96c** [comment]
Once those nits are sorted, ship it!

**Comment by AutomatedTester** [comment]
merged manually

**Pull request closed without merging**